lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -699,6 +699,10 @@ void applyTextStyleToElement({
   if (color != null) {
     cssStyle.color = colorToCssString(color);
   }
+  final double? strokeWidth = style.foreground?.strokeWidth;
+  if (strokeWidth != null) {
+    cssStyle.textStroke = '${strokeWidth}px ${colorToCssString(color)}';
+  }
```

Comment:
Same here. Check for `foreground.style == ui.PaintingStyle.stroke`.

Reply:
We fixed it, please check our comment for further information.